core/services/pipeline/task.base.go

```diff
@@ -0,0 +1,38 @@
+package pipeline
+
+import "time"
+
+type BaseTask struct {
```

Comment:
What does this task do?

Reply:
not a new task - just moved it into its own file